x/rollapp/types/message_create_rollapp.go

```diff
@@ -10,16 +10,19 @@ var _ sdk.Msg = &MsgCreateRollapp{}
 
 const MaxAllowedSequencers = 100
 
-func NewMsgCreateRollapp(creator string, rollappId string, maxSequencers uint64, permissionedAddresses []string,
-	metadatas []TokenMetadata, genesisAccounts []GenesisAccount,
+func NewMsgCreateRollapp(
```

Comment:
why we allow creating a rollapp with `TransfersEnabled=true`?

Reply:
I've now improved the doc
https://github.com/dymensionxyz/dymension/blob/961625f057abfaeff5c885d444fe878242e7f8b6/x/rollapp/types/tx.pb.go#L47-L49

Reply:
can't it be set to allowed on the first packet, where genesis transfers is empty?
in similar flow where u enable now

Reply:
hmm ok like
first transfer without a memo sets the bridge to enabled
and genesis transfers are only allowed if the bridge is disabled
I like it actually
but lets do in a followup PR?
needs to be changed in ADR as well